tools/ci/junit_upload.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# shellcheck source=/dev/null
+source /root/.bashrc
+# junit file name can differ in kitchen or macos context
+junit_files="junit-*.tar.gz"
+if [[ -n "$1" ]]; then
+    junit_files="$1"
+fi
+
+DATADOG_API_KEY="$("$CI_PROJECT_DIR"/tools/ci/aws_ssm_get_wrapper.sh "$API_KEY_ORG2_SSM_NAME")"
+export DATADOG_API_KEY
+errors=0
+for file in $junit_files; do
+    if [[ ! -f $file ]]; then
+        echo "Issue with junit file: $file"
+        continue
+    fi
+    inv -e junit-upload --tgz-path "$file" || ((errors++))
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏I think it would be proper to set `errors` to 1 because some exit codes might be ambiguous (127...)
```suggestion
    inv -e junit-upload --tgz-path "$file" || errors=1
```